pkg/importer/s3-datasource_test.go

```diff
@@ -135,7 +135,7 @@ var _ = Describe("S3 data source", func() {
 		sourceFile, err := os.Open(cirrosFilePath)
 		Expect(err).NotTo(HaveOccurred())
 
-		sd, err = NewS3DataSource("http://amazon.com", "", "")
+		sd, err = NewS3DataSource("http://region.amazon.com/bucket-1/object-1", "", "")
```

Comment:
Thanks for the PR!

I'm not very knowledgeable about S3, but needing to make this change makes me wonder if someone might be relying on syntax that will no longer work after change. Perhaps that should be documented in the release notes.

Reply:
Thanks for you comments. I have updated the release notes.